README.md

```diff
@@ -1,10 +1,253 @@
 
-==================
+The aim of the app is to provide a way for conference and event organizers to find and contact appropriate women* speakers. (But obviously there are many different contexts in which it can be used...)
+
+**Please note: Sometimes for better readability in long textures the term `women` is written without a star but we always mean everyone who defines as a woman.**
+
+
+# Technical Requirements
+
+- Ruby v2.2.7
+- Ruby on Rails, '~> 4.2.7.1 '
+- Elasticsearch v2.4.5 - https://www.elastic.co
+- ImageMagick - https://www.imagemagick.org
+- PostgreSQL v9.5.5 - https://www.postgresql.org
+- Bundler - https://bundler.io
+
+# Getting Started
+
+## 1. Copy & Configuration
+
+**1.1 Copy `config/database_example.yml` and create your own `database.yml` file and make sure it is always set to `.gitignore`:**
+
+```bash
+
+$ cp config/database_example.yml config/database.yml
+```
+
+**1.2 Adjust the settings of that config file according to your needs and ensure you have all server environments (development, test, staging, production) configured**
+
+
+## 2. Postgres
+
+**2.1 Install [PostgreSQL](http://www.postgresql.org/download/) dependent on your operating system**
+
+**2.2 Create a [PostgreSQL user](https://www.digitalocean.com/community/tutorials/how-to-use-roles-and-manage-grant-permissions-in-postgresql-on-a-vps--2) that is needed to log into the database (by default a `postgres` superuser is created after installation)**
+
+## 3. Elasticsearch
+
+**3.1 Install Elasticsearch 2.4**
+
+### Mac
+
+b) via homebrew
+
+```bash
+
+# Before Installation
+$ brew search elasticsearch # this will list all available Elasticsearch versions
+
+# Installation
+$ brew install elasticsearch@2.4
+
+# After installation
+$ echo 'export PATH="/usr/local/opt/elasticsearch@2.4/bin:$PATH"' >> ~/.zshrc # please check, if you have to replace `~/.zshrc` with any other dotfile, e.g. .bash_profile, .bashrc, etc.
+$ brew services list # check if Elasticsearch is now listed as brew services
+$ brew services start elasticsearch@2.4 # start elasticsearch
+
+# Troubleshooting:
+If you have any issues to install the old version (which is 'keg only'), try `brew link —force elasticsearch@2.4` to make Elasticsearch run.
+```
+
+### Ubuntu
+
+**3.2 c) Please follow the guide to install [Elasticseach 2.4.5](https://www.digitalocean.com/community/tutorials/how-to-install-and-configure-elasticsearch-on-ubuntu-16-04) and pay attention to the version!**
+
+```bash
+
+# Installation
+$ wget https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-2.4.5.deb
+$ sudo dpkg -i elasticsearch-2.4.5.deb
+$ sudo update-rc.d elasticsearch defaults
+
+# Check the Elasticsearch status
+$ sudo service elasticsearch status
+
+# Start Elasticsearch
+$ sudo service elasticsearch start
+```
+
+### Command Line / Download
```

Comment:
The curl command is still part of the ubuntu setup. You need it for getting the tests run. That curl programm downloads the elasticsearch version and moves it to the tmp folder in your app. So when running the tests they start that elasticsearch version.

Reply:
So maybe change that title to: 
**install elasticsearch for the rspec test**.?

Reply:
i think i am a bit confused here. :) do we have 2 different ubuntu ways to install elasticsearch (wget & curl) or is one only for the test setup?